easy_rec/python/train_eval.py

```diff
@@ -73,6 +74,9 @@ def main(argv):
       else:
         assert FLAGS.ignore_finetune_ckpt_error, 'fine_tune_checkpoint(%s) is not exists.' % FLAGS.fine_tune_checkpoint
 
+    if pipeline_config.fg_json_path:
+      fg_util.load_fg_json_to_config(pipeline_config)
```

Comment:
It’s better to load_fg_json_to_config in train_and_evaluate_impl.

Reply:
This is to make the priority of 'load_fg_json_to_config' less than 'load hpo_params' and 'edit_config_json'.